src/buildHTML.js

```diff
@@ -544,8 +544,11 @@ groupTypes.xArrow = function(group, options) {
     const arrowShift = -options.fontMetrics().axisHeight +
         0.5 * arrowBody.height;
     // 2 mu kern. Ref: amsmath.dtx: #7\if0#2\else\mkern#2mu\fi
-    const upperShift = -options.fontMetrics().axisHeight -
+    let upperShift = -options.fontMetrics().axisHeight -
         0.5 * arrowBody.height - 0.111;
+    if (group.value.label === "\\xleftequilibrium") {
+        upperShift -= upperGroup.depth;
+    }
```

Comment:
Is this defined in mhchem.sty?

Reply:
The short upwards harpoon has an interference that doesn't occur in the other extensible arrows. So I needed to bump the text up to miss the descender.

Reply:
In LaTeX/mhchem (= mhchem.sty), I used a different approach: I made the arrow longer, so that the descenders do not touch the arrow head. I am completely fine with your solution.